ds701_book/20-Recommender-Systems.qmd

```diff
@@ -127,174 +128,230 @@ Or it can be done passively:
 * Imbalanced dataset
     * User activity / item reviews are power law distributed    
 
-```{margin}
-This data is a subset of the data presented in: "From amateurs to connoisseurs: modeling the evolution of user expertise through online reviews," by J. McAuley and J. Leskovec. WWW, 2013
-```
+::: aside
```

Comment:
I think this should be on the following slide. There is no mention of a dataset on the challenges slide.

Reply:
This is generally true, not just for this data, so it's still relevant here.